.npmignore

```diff
@@ -0,0 +1,13 @@
+node_modules

```

Comment:
Since `package.json` includes a list of `files`, this isn't needed. `files` tells npm what to include when publishing. See more at https://docs.npmjs.com/files/package.json#files